src/liboslexec/llvm_util.cpp

```diff
@@ -5216,6 +5224,7 @@ LLVM_Util::op_store(llvm::Value* val, llvm::Value* ptr)
         std::cerr << "op_store val->getType()=" << std::flush;
         val->getType()->print(llvm::errs());
         std::cerr << std::endl;
+        // OSL_ASSERT(0);
```

Comment:
Remove commented out line(?)

Reply:
Wait, maybe I'll change it to DASSERT.

Reply:
I think my point was that it was currently commented out, so doing nothing.
Having it OSL_ASSERT or OSL_ASSERT is useful when in debugger because you get the builtin_trap and break right there.

Reply:
Updated to not-commented-out OSL_DASSERT, so it will hit the assertion in debug mode, but not for production builds.